contribute/anbox-style-guide.md

```diff
@@ -0,0 +1,86 @@
+* Use bold and italics very sparingly.Use italics for user interface fields and bold for UI elements that call for action.
+* Use single spaces between sentences.
+* Use angle brackets to indicate variables.
+* All extra white space should be removed, especially at the end of lines.
+* Add a new line at the end of the file.
+
+## Section titles
+
+* Insert a blank line after a section title.
+* Do not skip heading levels when creating sections.
+* Avoid using multiple section titles sequentially without any text between them.
+* Avoid manual HTML anchors, links to section titles can be picked up from the page-level table of contents.
+
+## Admonishments
+
+While admonishments can be useful to convey important information, they can be a distraction when used often. So before adding an admonishment, assess if the content really belongs in one.
+
+Keeping the types of admonishments to a minimum could be simplistic and also reduce cognitive overload on the reader. Stick to the following types of admonishments:
+
+* Note
+* Important
+* Tip
+* Caution
+
+## Language
+
+### Oxford/Serial Comma
+
+The Anbox Cloud documentation does not use the oxford comma but exceptions are allowed for the sake of technical accuracy and ease of readability.
+
+### Hyphens for Prefixes
+
+Use hyphens when the starting letter of a word is in capital case or starts with a vowel. In other cases, do not hyphenate and use it as a single word.
+
+    **Note:** You might be required to add such words to the wordlist as spellcheck could flag these words.
+
+**Examples:**
+* `Pre-employment`
+* `Preconfigure`
+
+## Terminology
+
+* It’s always “Anbox Cloud Appliance” when used in full form and “appliance” when used generically.
+* Always use product name in full form with capitalisation - "Anbox Cloud".
+* Within the context of Anbox Cloud, such as AMS, you can use the term "Anbox images" or "Anbox instances". When documenting about LXD images or containers that contain Anbox specific configuration from a big picture point of view,such as the architecture, use the term as "LXD images" or "LXD containers". When there is scope for ambiguity, elaborate and use as "LXD images with Anbox Cloud configuration".
+
+## Images and screenshots
+
+While images and screenshots are very helpful in visual appeal and getting the message across to the readers, they can be hard to maintain and cause overhead if they change too often. So use images or screenshots only when they:
+
+* Do not change often requiring updates for every release.
+* Explain the concept better than text or it should strongly reinforce the concept explained in the text.
+* Add to the reader’s understanding rather than just providing a step by step visual of the user interface.
+* Break the monotony of continuous text information when trying to accomplish a tough procedure.
+
+Simple images can be made using an image editor of your choice or you can use [diagrams.net](https://www.diagrams.net).
```

Comment:
(This is a bit unrelated to your changes)
I couldn't find anything specific for accessibility in the Ubuntu style guide.
In the context of images and screenshots, I see that it mentions alt-text. Do we have guidance on how to write alt-text for images for accessibility purposes?

Reply:
Some good resources I collected for now: https://webaim.org/techniques/alttext/ and https://design-system.service.gov.uk/styles/images/#alt-text

I will select appropriate guidelines and add this into our Anbox style guide but it will have to wait until later

Reply:
Thanks for the links!